[
  {
    "description": "",
    "elements": [
      {
        "description": "",
        "id": ";",
        "keyword": "Scenario",
        "line": 3,
        "name": "",
        "steps": [
          {
            "after": [
              {
                "match": {
                  "location": "support/ruby/env.rb:9"
                },
                "result": {
                  "duration": 228000,
                  "error_message": "Failing after step (RuntimeError)\n./support/ruby/env.rb:10:in `AfterStep'",
                  "status": "failed"
                }
              }
            ],
            "keyword": "Given ",
            "line": 4,
            "match": {
              "location": "support/ruby/env.rb:13"
            },
            "name": "this step passes",
            "result": {
              "duration": 52000,
              "status": "passed"
            }
          }
        ],
        "tags": [
          {
            "line": 2,
            "name": "@failing-after-step"
          }
        ],
        "type": "scenario"
      },
      {
        "description": "",
        "id": ";",
        "keyword": "Scenario",
        "line": 3,
        "name": "",
        "steps": [
          {
            "after": [
              {
                "match": {
                  "location": "support/ruby/env.rb:9"
                },
                "result": {
                  "duration": 930000,
                  "error_message": "Failing after step (RuntimeError)\n./support/ruby/env.rb:10:in `AfterStep'",
                  "status": "failed"
                }
              }
            ],
            "keyword": "Given ",
            "line": 4,
            "match": {
              "location": "support/ruby/env.rb:13"
            },
            "name": "this step passes",
            "result": {
              "duration": 47000,
              "status": "passed"
            }
          }
        ],
        "tags": [
          {
            "line": 2,
            "name": "@failing-after-step"
          }
        ],
        "type": "scenario"
      },
      {
        "description": "",
        "id": ";",
        "keyword": "Scenario",
        "line": 3,
        "name": "",
        "steps": [
          {
            "after": [
              {
                "match": {
                  "location": "support/ruby/env.rb:9"
                },
                "result": {
                  "duration": 130000,
                  "error_message": "Failing after step (RuntimeError)\n./support/ruby/env.rb:10:in `AfterStep'",
                  "status": "failed"
                }
              }
            ],
            "keyword": "Given ",
            "line": 4,
            "match": {
              "location": "support/ruby/env.rb:13"
            },
            "name": "this step passes",
            "result": {
              "duration": 24000,
              "status": "passed"
            }
          }
        ],
        "tags": [
          {
            "line": 2,
            "name": "@failing-after-step"
          }
        ],
        "type": "scenario"
      }
    ],
    "id": "",
    "keyword": "Feature",
    "line": 1,
    "name": "",
    "uri": "examples/single/failing-after-step-hook.feature"
  }
]
